keyboards/dztech/dz60v2/info.json

```
@@ -0,0 +1,2027 @@
+{
+    "manufacturer": "DZTECH",
+    "keyboard_name": "DZ60",
+    "maintainer": "DZTECH",
+    "bootloader": "atmel-dfu",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "E6",
+        "on_state": 1
+    },
+    "matrix_pins": {
+        "cols": ["F1", "D0", "D1", "D2", "D3", "D5", "D4", "D6", "D7", "B4", "F7", "C7", "C6", "B6", "B5"],
+        "rows": ["F0", "B3", "F6", "F4", "F5"]
+    },
+    "processor": "atmega32u4",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+
+    },
+        "brightness_steps": 10,
+        "hue_steps": 10,
+        "led_count": 16,
+        "pin": "B2",
+        "saturation_steps": 10
+    },
```

assuming RGB lighting will use the ws2182 driver, defined pin will need to be moved to `ws2812.pin`
```suggestion
        "saturation_steps": 10
    },
    "ws2812": {
        "pin": "B2"
    },
```